plugins/com.google.cloud.tools.eclipse.appengine.flex.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,18 @@
+Manifest-Version: 1.0
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: Front End App Engine Flexible Support Tests
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.flex.test
+Bundle-Version: 0.1.0.qualifier
```

Comment:
could we move this to 1.0.0 or would that just break everything and be a huge ball of mess?